public/app/engagementindex/leaderboard/leaderboard.html

```diff
@@ -7,7 +7,7 @@
 <div data-ng-if="!me.is_admin && me.share_points !== null" data-ng-include="'/app/engagementindex/leaderboard/userinfo.html'"></div>
 
 <!-- LIST -->
-<div data-ng-if="me.is_admin || me.share_points" data-ng-include="'/app/engagementindex/leaderboard/list.html'"></div>
+<div data-ng-include="'/app/engagementindex/leaderboard/list.html'"></div>
```

Comment:
This causes the `Points configuration` button to also be present on the `opt in/out` screen students see when to go the Engagement Index for the first time. The `Points configuration` button should not be visible there.